source/views/tasks_view.py

```diff
@@ -164,3 +166,14 @@ def get_statuses(self):
         statuses_data = self.status_schema.dump(statuses, many=True).data
         return jsonify({'statuses':statuses_data})
 
+
+    def get_users_per_task(self, id_task):
+        authorization(request.headers.get('Authorization', None))
+        collaborators = TaskOwner.query.filter_by(id_task=id_task)\
+            .filter(TaskOwner.id_task_owner!=TaskOwner.id_user).all()
+        collaborators_list = list()
+        for collaborator in collaborators:
+            task_user = User.query.get(collaborator.id_user)
```

Comment:
Change the "get" with get_or_404, this will generate an 404 mistake if the user was not found with that id.